src/components/Reports/StaffExpenseReport/DownloadButtonGroup/downloadReport.tsx

```diff
@@ -0,0 +1,74 @@
+import { DateTime } from 'luxon';
+import { Transaction } from 'src/components/Reports/StaffExpenseReport/StaffExpenseReport';
+import { ReportType } from '../Helpers/StaffReportEnum';
+
+const csvHeader = ['Date', 'Category', 'Amount'];
+
+const makeTable = (
+  title: string,
+  transactions: Transaction[],
+  type: ReportType,
+) => {
+  if (type === ReportType.Income) {
```

Comment:
Can you make a separate variable for the filtered transactions instead of reassigning `transactions`?